eng/pipelines/runtime.yml

```diff
@@ -75,132 +102,354 @@ jobs:
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - Windows_NT_x86
+    - ${{ if eq(variables['isFullMatrix'], true) }}:
+      - Windows_NT_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    jobParameters:
+      isFullMatrix: ${{ variables.isFullMatrix }}
+      framework: netfx
+      runTests: true
+      testScope: innerloop
+      condition: or(eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true), eq(variables['isFullMatrix'], true))
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+    platforms:
+    - Windows_NT_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    jobParameters:
+      isFullMatrix: ${{ variables.isFullMatrix }}
+      framework: allConfigurations
+      runTests: true
+      liveCoreClrBuildConfig: release
+      condition: or(eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true), eq(variables['isFullMatrix'], true))
+
+#
+# Installer Build and Test
+# These are always built since they only take like 15 minutes
+# we expect these to be done before we finish libraries or coreclr testing.
+#
+- template: /eng/pipelines/installer/installer-matrix.yml
+  parameters:
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+    platforms:
+      - Linux_arm
+      - Linux_musl_arm64
+      - Windows_NT_x86
+      - Windows_NT_arm
+      - Windows_NT_arm64
+    jobParameters:
+      liveCoreClrBuildConfig: release
+      liveLibrariesBuildConfig: Release
+
+- template: /eng/pipelines/installer/installer-matrix.yml
+  parameters:
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+    platforms:
+      - OSX_x64
+      - Linux_x64
+      - Linux_arm64
+      - Linux_musl_x64
+      - Windows_NT_x64
+    jobParameters:
+      liveCoreClrBuildConfig: release
+      liveLibrariesBuildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+
+#
+# Libraries Test Build
+# Only when CoreCLR or Libraries is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-test-job.yml
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+    platforms:
+    - OSX_x64
+    - Linux_x64
+    - Windows_NT_x64
     jobParameters:
       isOfficialBuild: false
-      liveCoreClrBuildConfig: checked
+      liveCoreClrBuildConfig: release
+      testScope: innerloop
+      condition: or(eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true), eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true), eq(variables['isFullMatrix'], true))
```

Comment:
I'm pretty sure you can split this across lines using `>-` for readability:

```yaml
      condition: >-
        or(
          eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
          eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
          eq(variables['isFullMatrix'], true))
```

Reply:
Thanks, I also wanted to achieve that but I didn't know we could do it for conditions 😄-- will do it, I also hate horizontal scrolling but didn't know I could do it in this case.

Reply:
IIRC AzDO balks at newlines in the string (when trying to do this without `>-`), but luckily yaml has features for us. 🙂 http://yaml-multiline.info/